.github/pull_request_template.md

```diff
@@ -6,6 +6,20 @@
 
 (Details)
 
+## Does this PR affect the documentation?
```

Comment:
minor: What about using "update" or "change" instead of "affect"? That would be more explicit :)

Reply:
"change" sounds good to me :)